include/MiniLua/environment.hpp

```diff
@@ -54,14 +52,20 @@ class Environment {
      * Add multiple variables to the environment.
      */
     void add_all(std::unordered_map<std::string, Value> values);
-    void add_all(std::initializer_list<std::pair<const std::string, Value>> values);
+    void add_all(std::initializer_list<std::pair<std::string, Value>> values);
```

Comment:
What is the reason that you removed the `const` from the string?

Reply:
I don't remember but it was probably because it wasn't needed and the other `add_all` methods also didn't have it. The `initializer_list` version of the method is also only called when you specify the parameter in the code directly. I.e. `env.add_all({{"key1", 5}, {"key2", 42}});`. So I'm not sure if it would even matter.

Reply:
If there are problem we can add it again (or maybe add another overload). But I think it should not change any behavior.